CMakeLists.txt

```diff
@@ -137,15 +137,22 @@ endif(FAULT_INJECTION)
 #add_definitions(-DBOOST_LOG_DYN_LINK)
 
 if(NOT CMAKE_BUILD_TYPE)
-    message(STATUS "No CMAKE_BUILD_TYPE specified, defaulting to Valgrind")
-    set(CMAKE_BUILD_TYPE Valgrind)
+    message(STATUS "No CMAKE_BUILD_TYPE specified, defaulting to ${CMAKE_BUILD_TYPE}")
```

Comment:
Should we default to something else then? Or just print a warning, possibly with the available build types?

In any case, this message looks funny as `${CMAKE_BUILD_TYPE}` should be empty there.

Reply:
But it's not empty, it's `Release`.

Reply:
oops, looks like I was doing something wrong, it's empty indeed.